tests/NuGet.Services.AzureSearch.FunctionalTests/Support/NuGetSearchFunctionalTestBase.cs

```diff
@@ -56,7 +56,7 @@ public NuGetSearchFunctionalTestBase(CommonFixture fixture, ITestOutputHelper te
 
             if (take.HasValue)
             {
-                requestUri += $"&skip={take}";
+                requestUri += $"&take={take}";
```

Comment:
This didn't break our tests before as the second `skip` query parameter was ignored